rocketpy/Rocket.py

```diff
@@ -428,7 +489,167 @@ def evaluateStaticMargin(self):
         self.staticMargin.setDiscrete(
             lower=0, upper=self.motor.burnOutTime, samples=200
         )
-        return None
+        # Return self
+        return self
+
+    def evaluateDryInertias(self):
+        """Calculates and returns the rocket's dry inertias relative to
+        the rocket's center of mass. The inertias are saved and returned
+        in units of kg*m².
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        -------
+        self.dry_I_11 : float
```

Comment:
is it really a float?

Please check it because it smells like rocketpy.Function again

Reply:
Yes, all dry inertias should be floats